Beta build doesn't launch from console

### Description
Beta build doesn't launch from console

### Steps to Reproduce
   1. Install 0.20.6
   2. Close the browser 
   3. Launch from console, doesn't launch, same result as #11791, console shows error 

**Actual result:**
```
$ An uncaught exception occurred in the main process Uncaught Exception:
Error: Cannot find module './updater'
    at Module._resolveFilename (module.js:470:15)
    at Function.Module._resolveFilename (C:\Users\Sriram BV\AppData\Local\BraveBeta\app-0.20.6\resources\electron.asar\common\reset-search-paths.js:41:12)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at C:\Users\Sriram BV\AppData\Local\BraveBeta\app-0.20.6\resources\app.asar\app\index.js:53:17
    at Object.<anonymous> (C:\Users\Sriram BV\AppData\Local\BraveBeta\app-0.20.6\resources\app.asar\app\index.js:356:2)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
[1107/145650.716:ERROR:process_info.cc(632)] range at 0x9e864b0600000000, size 0x1f3 fully unreadable
[1107/145650.716:ERROR:process_info.cc(632)] range at 0x9e864b2600000000, size 0x1f3 fully unreadable
[1107/145650.716:ERROR:process_info.cc(632)] range at 0x0, size 0x1f3 fully unreadable
Waiting 60 seconds for process to load
```

**Expected result:**
Browser should launch from console

**Reproduces how often:**
100%

### Brave Version
**about:brave info:**

Brave | 0.20.6
-- | --
V8 | 6.2.414.36
rev | 18f7d1e
Muon | 4.5.14
OS Release | 10.0.15063
Update Channel | Beta
OS Architecture | x64
OS Platform | Microsoft Windows
Node.js | 7.9.0
Brave Sync | v1.4.2
libchromiumcontent | 62.0.3202.75

**Reproducible on current live release:**
N/A


### Additional Information

--- a/app/windowsInit.js
+++ b/app/windowsInit.js
@@ -76,13 +76,16 @@ if (process.platform === 'win32') {
   }
 
   const userDataDirSwitch = '--user-data-dir-name=brave-' + channel
-  if (channel && channel !== 'dev' && !process.argv.includes(userDataDirSwitch)) {
+  if (channel !== 'dev' && !process.argv.includes(userDataDirSwitch) &&
+      !process.argv.includes('--relaunch') &&
+      !process.argv.includes('--user-data-dir-name=brave-development')) {
+    delete process.env.CHROME_USER_DATA_DIR
     if (cmd === '--squirrel-firstrun') {
       app.relaunch({args: [userDataDirSwitch, '--relaunch']})
     } else {
-      app.relaunch({args: process.argv.slice(1) + [userDataDirSwitch, '--relaunch']})
+      app.relaunch({args: process.argv.slice(1).concat([userDataDirSwitch, '--relaunch'])})
     }
-    app.quit()
+    app.exit()
   }
 }
 

--- a/tools/buildInstaller.js
+++ b/tools/buildInstaller.js
@@ -178,9 +178,7 @@ if (isDarwin) {
       setupExe: `${appName}-Setup-${arch}.exe`
     })
     resultPromise.then(() => {
-      cmds = [
-      ]
-      execute(cmds, {}, console.log.bind(null, 'done'))
+      console.log('done')
     }, (e) => console.log(`No dice: ${e.message}`))
   })
 } else if (isLinux) {

